### ISSUE TITLE
@intlify/vue-i18n/no-raw-text  should consider the case of variable substitution

### ISSUE THREAD

**Issue by fshipei** [intlify/eslint-plugin-vue-i18n#320]
**Please describe what the rule should do:**
<img width="209" alt="截屏2022-04-18 下午11 55 41" src="https://user-images.githubusercontent.com/39074563/163835624-527d9b88-e3c0-43f4-8382-e0e76f002a73.png">

@intlify/vue-i18n/no-raw-text  should consider the case of variable substitution like this, and even cases where parent and child components pass values

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[ ] Warns about a potential error (problem)

[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue

```

```vue

```

```vue

```

**Additional context**

**Comment by ota-meshi** [comment]
I'm not sure what you want from the rule.  Write each section of the template.
